### PR TITLE
OSRA428 - Added integration test for CRUD partner

### PR DESCRIPTION
**Pull request by gmunumel**
Added integration test for CRUD partner: https://osraav.atlassian.net/browse/OSRA-428

### CODE REVIEW

File: spec/features/hq/crud_partner_spec.rb [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    given_partners_exist [ { name: 'Partner1', region: 'Region1', status: 'Active', province: 'Damascus & Rif Dimashq', contact_details: 12345, start_date: '2013-09-25' },
+                           { name: 'Partner2', region: 'Region2', status: 'Active', province: 'Aleppo',                 contact_details: 23456, start_date: '2013-09-25' },
+                           { name: 'Partner3', region: 'Region3', status: 'Active', province: 'Homs',                   contact_details: 34567, start_date: '2013-09-25' } ]
+    i_sign_in_as_admin
+  end
+
+  scenario 'There should be a link to the partners page on the navbar' do
+    visit hq_root_path
+    expect(page).to have_link('Partners', href: hq_partners_path)
+  end
```
Comment (razorcd) [inline spec/features/hq/crud_partner_spec.rb]: this scenario is already tested in the navigation feature. Please remove it.

**Comment by razorcd** [comment]
please rebase this and leave a comment

**Comment by gmunumel** [comment]
I have forgot to do this. I'll try to do it this weekend. Sorry.

**Comment by gmunumel** [comment]
@razorcd Done, everything good :+1:

**Pull request merged**